desktop-src/SysInfo/converting-a-time-t-value-to-a-file-time.md

```diff
@@ -1,32 +1,42 @@
 ---
-description: The time functions included in the C run-time use the time\_t type to represent the number of seconds elapsed since midnight, January 1, 1970. The following example converts a time\_t value to a file time, using the Int32x32To64 function.
+description: The time functions included in the C run-time use the **time_t** type to represent the number of seconds elapsed since midnight, January 1, 1970. The following example converts a **time_t** value to a [FILETIME](/windows/win32/api/minwinbase/ns-minwinbase-filetime).
 ms.assetid: f626c0b2-a5a1-475d-9a24-64e7b0407278
-title: Converting a time_t Value to a File Time
+title: Converting a time_t value to a FILETIME
 ms.topic: article
-ms.date: 05/31/2018
+ms.date: 12/03/2021
 ---
 
-# Converting a time\_t Value to a File Time
+# Converting a time\_t value to a FILETIME
 
-The time functions included in the C run-time use the time\_t type to represent the number of seconds elapsed since midnight, January 1, 1970. The following example converts a time\_t value to a file time, using the [**Int32x32To64**](/windows/desktop/api/winnt/nf-winnt-int32x32to64) function.
+The time functions included in the C run-time use the **time_t** type to represent the number of seconds elapsed since midnight, January 1, 1970. The following example converts a **time_t** value to a [FILETIME](/windows/win32/api/minwinbase/ns-minwinbase-filetime).
 
-
-```C++
+```cpp
 #include <windows.h>
 #include <time.h>
 
-void TimetToFileTime( time_t t, LPFILETIME pft )
+void TimetToFileTime(time_t t, LPFILETIME pft)
 {
-    LONGLONG ll = Int32x32To64(t, 10000000) + 116444736000000000;
-    pft->dwLowDateTime = (DWORD) ll;
-    pft->dwHighDateTime = ll >>32;
+    ULARGE_INTEGER time_value;
+    time_value.QuadPart = (t * 10000000LL) + 116444736000000000LL;
+    pft->dwLowDateTime = time_value.LowPart;
+    pft->dwHighDateTime = time_value.HighPart;
 }
 ```
 
+After you've a [FILETIME](/windows/win32/api/minwinbase/ns-minwinbase-filetime), you can convert the value to system time using the [**FileTimeToSystemTime**](/windows/win32/api/timezoneapi/nf-timezoneapi-filetimetosystemtime) function.
```

Comment:
Missing "obtained" from the sentence.

Reply:
Good catch, @CookiePLMonster. I've fixed that in a new PR into the private repo. Ordinarily, that would be live within an hour or so, but we have an issue with the publishing pipeline. That should be cleared early next week, so both this PR and the one I just created should be live then.

Thanks!
Steve